.github/workflows/ubuntu.yml

```diff
@@ -55,7 +55,6 @@ jobs:
           PREFIX: ${{ github.repository }}_
           DOCKER_IMAGE: ghcr.io/tesseract-robotics/tesseract_ros2:${{ matrix.distro }}-master
```

Comment:
```suggestion
          DOCKER_IMAGE: ghcr.io/tesseract-robotics/tesseract_ros2:${{ matrix.distro }}-0.18
```

Reply:
Looks like there currently isn't tagged versions of the docker being produced by tesseract_ros2. Also, this works with the latest version of Tesseract any way.

Reply:
Yeah, I recently updated `tesseract_ros2` CI to build on releases, so we just need to create a tag in that repo to trigger a job that makes this docker image. I'm fine to merge this now and fix it later if you want, or we can do this first